nodes/cma/nodes-cma-autoscaling-custom-trigger-auth.adoc

```diff
@@ -25,20 +39,20 @@ metadata:
   namespace: my-namespace <1>
 spec:
   secretTargetRef: <2>
-  - parameter: user-name <3>
+  - parameter: username <3>
     name: my-secret <4>
-    key: USER_NAME <5>
+    key: username <5>
   - parameter: password
     name: my-secret
```

Comment:
This needs to be `keda-prom-secret` now since it's using the new example secret.